main.go

```diff
@@ -47,22 +53,26 @@ func main() {
 
 		login := &cobra.Command{
 			Use:   "login",
-			Short: "Authenticate with an Atomist workspace",
+			Short: "Authenticate with Atomist workspace",
 			RunE: func(cmd *cobra.Command, _ []string) error {
-				if !query.CheckAuth(workspace, apiKey) {
-					return errors.New("login failed")
-				} else {
+				apiKey, err := readApiKey(apiKeyStdin, dockerCli)
+				if err != nil {
+					return err
+				}
+				if query.CheckAuth(workspace, apiKey) {
```

Comment:
Not urgent, but I noticed that running `docker base login` without any flags set would produce an error as `workspace` was a required flag.

```bash
docker base login
required flag(s) "workspace" not set
```

From a usability perspective, it would be nice if in that case it would first prompt for the workspace, then ask for the API key, similar to `docker login`;

```bash
docker login
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.
Username:
```

It's common to use flags for _optional_ arguments, and positional arguments as _required_ arguments, so if it's a _required_ parameter, then it could be `docker base login <workspace>`, but of course if it would be interactive, then `--workspace` would work for that.

Reply:
Ha, good point. I'll change that.